analyticskit/build.gradle

```diff
@@ -47,6 +47,7 @@ android {
 
 dependencies {
 	api "androidx.annotation:annotation:1.2.0"
+	compileOnly "org.jetbrains.kotlin:kotlin-stdlib:$rootProject.ext.kotlinVersion"
```

Comment:
you should be able to just put this on API. with androidx being kotlin first I dare you to find me an app that doesn't already have the kotlin std lib included.